### PR TITLE
Updated defaults

### PR DESCRIPTION
**Pull request by Craz1k0ek**
* Added some more defaults settings

### CODE REVIEW

**Review by Enzime — CHANGES_REQUESTED** [review]
Can you update the tests?

File: modules/system/defaults/finder.nix [hunk 1]
```diff
@@ -38,6 +38,15 @@ with lib;
       '';
     };
 
+    system.defaults.finder.FXRemoveOldTrashItems = mkOption {
+      type = types.nullOr types.bool;
+      default = null;
+      description = ''
+        Remove items in the bin after 30 days.
```
Comment (Samasaur1) [inline modules/system/defaults/finder.nix:45]: Should the docs call it the trash instead of the bin?

**Comment by Craz1k0ek** [comment]
> Can you update the tests?

@Enzime Not sure what I need to update to make the test succeed. Error message is incomprehensible to me

**Comment by Enzime** [comment]
Can you rebase your PR on master? It should fix the failing test, and then can you add some more tests to test the new options you added?

**Comment by Craz1k0ek** [comment]
@Enzime not sure why the tests are failing. Anything wrong in my syntax? The error message is so poor, I can't make anything of it

**Comment by Enzime** [comment]
This has been superseded by #1163

**Pull request closed without merging**